stratum/src/lib.rs

```diff
@@ -496,4 +486,44 @@ mod tests {
             response
         );
     }
+
+    #[test]
+    fn can_respond_to_submition() {
```

Comment:
I doubt if we can call this test as "unit test".
Usually scenarios included in source code is for unit testing, but this one seems to test operations on real world environment(e.g. opening tcp stream).
I think it'd be better test this kind of operations on integration tests.

Reply:
I have added a new one to existing test.
But I agree with you, so I will apply this to an integrated test.

Reply:
I actually meant that we should remove this test from this file..... but it seems there already exists similar test cases here.
I think we should file an issue about this problem, and revisit this issue later.

Reply:
Filed an issue at #457

Reply:
@Kais-DkM Thank you, I checked #457 out